rinks/cabinjohn.html

```diff
@@ -0,0 +1,388 @@
+								<li><a href="#"> Wheaton Ice Rink </a></li>
+
+							</ul>
+						</div> -->
+
+
+						</li>
+
+
+
+            <li><a href="../map.html">Map</a></li>
+            <li><a href="#">About</a></li>
+            <li><a href="#">Contact Us</a></li>
+          </ul>
+        </div>
+
+      </div>
+    </header>
+
+		<div class = "container" "clearfix">
+
+			<div class = "left_col">
+
+				<div class="info">
+					<h1>Cabin John Ice Rink</h1>
+					<a href="http://www.montgomeryparks.org/enterprise/ice/cabin_john/", target="new">http://www.montgomeryparks.org/cabin_john</a>
+				</div>
+
+
+					<div class="rinkmap">
+						<iframe src="https://www.google.com/maps/embed?pb=!1m14!1m8!1m3!1d12397.302233183442!2d-77.14824!3d39.030696!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x0000000000000000%3A0x8669ec2ef9c22378!2sCabin+John+Ice+Rink!5e0!3m2!1sen!2sus!4v1439137775305" width="300" height="150" frameborder="0"  style="border:0" allowfullscreen>
+						</iframe>>
+					</div>
+
+					<div class="rinkAddress">
+
+						<p>627 N Glebe Rd Suite 800</p>
+						<p>Rockville, MD 20852 | USA</p>
+						<p>(301) 765-8620</p>
+
+					</div>
+
+					<div class="actions">
+
+						<a href="#"> Favorite this rink </a>
+						<a href="#"> Print full schedule </a>
+						<a href="#"> Share this page </a>
+
+					</div>
+
+					<div class="nearByRinks">
+						<h5> Nearby rinks </h5>
+						<ul>
+
+						<li>	<a href="kettler.html"> Kettler Capitals Iceplex </a> </li>
+						<li>	<a href="rockville.html"> Rockville Ice Areana </a> </li>
+						<li>	<a href="reston.html"> Skatequest </a> </li>
+						<li>	<a href="wheaton.html"> Wheaton Ice Rink </a> </li>
+
+						</ul>
+					</div>
+
+
+
+
+
+			</div>
+
+			<div class="right_col">
+
+					<div class="featureIce clearfix"> <!--Daily Schedule-->
+
+						<div class="today">
+							September 1, 2015
+						</div>
+
+							<div class="featureTime"><!--Pick Up Times-->
+
+
+
+									<div class="headline">
+										Pick-Up Schedule
+									</div>
+
+									<div class="iceTime">
+										<h1>N/A</h1>
+										<!-- <h2>12 skaters + 1 goalie(s) registered</h2>
+										<a href="#">Sign-up</a> -->
+									</div>
+
+							</div>
+
+							<div class="featureTime"><!--SnS Times-->
+
+								<div class="headline">Stick 'n' Shoot Schedule</div>
+
+								<div class="iceTime">
+									<h1>8:30am - 10:00am</h1>
+									<h2>3 skaters + 0 goalie(s) registered</h2>
+									<a href="#">Sign-up</a>
+								</div>
+
+								<div class="iceTime">
+									<h1>10:30am - 12:00pm</h1>
+									<h2>5 skaters + 0 goalie(s) registered</h2>
+									<a href="#">Sign-up</a>
+								</div>
+
+								<div class="iceTime">
+									<h1>2:00pm - 3:00pm</h1>
+									<h2>10 skaters + 1 goalie(s) registered</h2>
+									<a href="#">Sign-up</a>
+								</div>
+
+								<div class="iceTime">
+									<h1>3:15pm - 4:45pm</h1>
+									<h2>16 skaters + 2 goalie(s) registered</h2>
+									<a href="#">Sign-up</a>
```

Comment:
As you already know, the real challenge will be in dynamically grabbing and populating this data, instead of hard-coding it.